aries_cloudagent/resolver/default/legacy_peer.py

```diff
@@ -94,7 +81,9 @@ class LegacyDocCorrections:
           "type": "did-communication",
           "priority": 0,
           "recipientKeys": ["did:sov:JNKL9kJxQi5pNCfA8QBXdJ#1"],
-          "routingKeys": ["9NnKFUZoYcCqYC2PcaXH3cnaGsoRfyGgyEHbvbLJYh8j"],
+          "routingKeys": [
+              "did:key:z6Mknq3MqipEt9hJegs6J9V7tiLa6T5H5rX3fFCXksJKTuv7#z6Mknq3MqipEt9hJegs6J9V7tiLa6T5H5rX3fFCXksJKTuv7"
```

Comment:
Will changing the routing key have any impact on existing communications?

Reply:
This LegacyDocCorrections class added in #2404 will handle translating the `routingKeys` to did:keys from raw base58 encoded keys. This correction here was just a typo where I forgot to change the routingKeys when I originally pasted in the example. The tests for these corrections were correctly exercising the code, just a matter of docs lagging behind implementation. :slightly_smiling_face: